example/generated_examples_test.go

```diff
@@ -463,6 +464,31 @@ func TestPaymentMethodDetach(t *testing.T) {
 	assert.NotNil(t, result)
 }
 
+func TestCardUpdate(t *testing.T) {
```

Comment:
We should skip these 4 generated tests.

We don't currently set either a `customer` or `account` in these, so the call fails.

You can verify this by changing say:

```
	result, _ := card.Update("ba_xxxxxxxxxxxxx", params)
```

to

```
	result, err := card.Update("ba_xxxxxxxxxxxxx", params)
```

and printing out `err`:

```
Invalid card params: either Customer or Account need to be set
```

Our underlying test manifest should probably be updated to have these cases, but in the mean time we'll rely on the more fleshed out tests in `card/client_test.go` which does set these values.